composer.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "flits/bik-sdk",
+  "description": "An Bik-sdk for php project",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Flits",
+      "email": "support@getflits.com"
+    }
+  ],
+  "require": {
+    "php": ">=5.6.0",
+    "guzzlehttp/guzzle": "^7.0"
+  },
+  "autoload": {
+    "psr-4": {
+      "Flits\\Bik\\": "src/"
+    }
+  },
+  "extra": {
+    "branch-alias": {
+      "dev-master": "1.0-dev"
```

Comment:
What is purpose of this?

Reply:
I just used it from earlier repositories